.github/workflows/build-release.yml

```diff
@@ -11,6 +17,16 @@ jobs:
         uses: actions/checkout@v3.1.0
       - name: Install composer dependencies
         run: composer install --no-dev -o
+      - name: Custom clean via Makefile
+        if: ${{ hashFiles('Makefile') != '' && inputs.makefile_rule != '' }}
```

Comment:
What do you think of this?

```suggestion
        if: ${{ inputs.makefile_rule != '' }}
```

This warns immediately the maintainer that something is missing on its repository, in this case the Makefile.